lib/node.js

```diff
@@ -687,7 +698,28 @@ declare module "http" {
 }
 
 declare module "https" {
-  // TODO
+  declare class Server extends tls$Server {
+    listen(port: number, hostname?: string, backlog?: number, callback?: Function): Server;
+    listen(path: string, callback?: Function): Server;
+    listen(handle: Object, callback?: Function): Server;
+    close(callback?: Function): Server;
+    setTimeout(msecs: number, callback: Function): Server;
+    timeout: number;
+  }
+
+  declare class ClientRequest extends http$ClientRequest {}
+  declare class IncomingMessage extends http$IncomingMessage {}
+
+  declare function createServer(options: Object, secureConnectionListener?: function): typeof Server;
```

Comment:
`typeof Server` returns the `Server` class, not an instance of `Server`. I think you want to drop the `typeof`?